mod_reforged/hooks/config/strings.nut

```diff
@@ -1257,6 +1270,20 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
  	}),
+	RF_HiddenPockets = ::UPD.getDescription({
+		Effects = [{
```

Comment:
I'd improve the formatting here by pushing the `{` to the next line and similarly at the end pushing the `]` to the next line. Then the tables can be indented one tab to the right.